lib/Mojolicious.pm

```diff
@@ -425,6 +425,9 @@ will allow requests of indefinite size. Note that increasing this value can also
 should you for example attempt to parse an excessively large request body with the methods L<Mojo::Message/"dom"> or
 L<Mojo::Message/"json">.
 
+If a request exceeds this limit it may be truncated, but your action will still process. Your action should test for
+requests which exceed this limit with L<Mojo::Message/"is_limit_exceeded">.
+
```

Comment:
That's a guide style description, this seems like the wrong place for it.

Reply:
Suggestions for the right place welcome.

Reply:
Additionally we will likely change how this is handled in the future. Parser errors ideally shouldn't reach the router in the first place. So new documentation is probably counter productive in that regard.

Reply:
Heh! I got here via the faulty assumption that exceeding the max_request_size would just give a 413 and no service like Apache's LimitRequestBody. Cancelling this request for reasons given here: https://github.com/mojolicious/mojo/issues/1620